index.html

```diff
@@ -0,0 +1,2059 @@
+                    class="card-img-top"
+                    alt="walmart-store-front"
+                /></a>
+                <a href="#"
+                  ><span class="wrap font-weight-bold badge-secondary px-3"
+                    >BUSINESS</span
+                  ></a
+                >
+              </div>
+              <div class="">
+                <h5 class="card-title font-weight-bold pt-3">
+                  <a class="h5-a" href="#"
+                    >Trump Compares Himself to 'Great Chess Master' in Talks
+                    With Putin, Xi, Kim</a
+                  >
+                </h5>
+                <p class="card-text">
+                  The president spoke of the world leaders being "very sharp"
+                  and "at the top of their game" when discussing the
+                  difficulties of negotiating with them.
+                </p>
+              </div>
+            </article>
+          </div>
+          <!--End of featured stories-->
+          <!--Debate & Opinion section-->
+          <section class="d-md-none gap">
+            <div class="debate py-4">
+              <div class="h2-div font-weight-bold">THE DEBATE</div>
+              <div class="line"></div>
+              <div class="media the-debate">
+                <img
+                  class="d-flex mr-3"
+                  alt="Major"
+                  src="https://d.newsweek.com/en/full/1621698/major-neill-franklin-ret.png?w=150&h=150&f=fedb525be44c92fd2598c72f563e8f95"
+                  style="width: 64px; height: 64px;"
+                />
+                <div class="media-body">
+                  <h5 class="mt-0 font-weight-bold">
+                    Legalization Is the Only Way to Improve the Criminal Justice
+                    System
+                  </h5>
+                  <p>
+                    BY<span class="span-debate font-weight-bold">
+                      MAJOR NEILL FRANKLIN (RET.)</span
+                    >
+                  </p>
+                </div>
+              </div>
+              <div class="vs-container">
+                <div class="line"></div>
+                <div class="vs">VS</div>
+              </div>
+              <div class="media">
+                <div class="media-body">
+                  <h5 class="mt-0 mb-1 font-weight-bold">
+                    Legalizing Pot Is a Catastrophic Mistake
+                  </h5>
+                  <p>
+                    BY
+                    <span class="span-debate font-weight-bold"
+                      >KEVIN SABET</span
+                    >
+                  </p>
+                </div>
+                <img
+                  class="d-flex ml-3"
+                  src="https://d.newsweek.com/en/full/111059/kevin-sabet.png?w=150&h=150&f=2d885279d159e97deb11590f95b67c10"
+                  alt="Kevin Sabet"
+                  style="width: 64px; height: 64px;"
```

Comment:
- [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)